.jenkins/tests/pom.xml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
I don't think this project (OptaPlanner) should own definition of dependencies, and their versions, for testing jenkinsfiles. This `pom.xml` brings a maintenance cost, which will be repeated across any repository that includes such tests.
We've already touched this topic with @radtriste and created https://issues.redhat.com/browse/BXMSPROD-1125.

Reply:
CC @ge0ffrey

Reply:
I guess we should remove all the maven related stuff. but leave the test files for now, it will be reused in https://issues.redhat.com/browse/BXMSPROD-1125.
@rsynek wdyt ?

Reply:
@Kevin-Mok I guess you should remove the tests part. will be handled later